Ruler ignore users (#4074)
cortexproject/cortex: docs/configuration/config-file-reference.md

# Enable the ruler api
# CLI flag: -experimental.ruler.enable-api
[enable_api: <boolean> | default = false]

# Comma separated list of tenants whose rules this ruler can evaluate. If
# specified, only these tenants will be handled by ruler, otherwise this ruler
# can process rules from all tenants. Subject to sharding.
# CLI flag: -ruler.enabled-tenants
[enabled_tenants: <string> | default = ""]

> # Comma separated list of tenants whose rules this ruler cannot evaluate. If
> # specified, and ruler would normally pick given tenant for processing (eg. via
> # sharding), tenant will be ignored instead.
> # CLI flag: -ruler.disabled-tenants

Reviewer: ```suggestion
# Comma separated list of tenants whose rules this ruler cannot evaluate. If
# specified, a ruler that would normally pick the specified tenant(s) for processing
# will ignore them instead. Subject to sharding. 
# CLI flag: -ruler.disabled-tenants
```

Reviewer: The homogenous wording indicates that this process is the exact opposite of the above. Differences in words here would leave an operator puzzled on whenever this means something different. WDYT? 

Feel free to move this to the where this text is generated.

Author: I will use your wording. Thanks for suggestion.
